Source/Fuse.Controls.Panels/Backgrounds/TopFrameBackground.uno

```diff
@@ -51,8 +51,10 @@ namespace Fuse.Controls
 		{
 			if defined(iOS || Android)
 			{
-				var x = SystemUI.TopFrame.Size / AppBase.Current.PixelsPerPoint;
-				return x;
+				var pixelsPerPoint = 1.0f;
```

Comment:
Hmm, this is a `visual` so it should actually be using the `Viewport` to get access to this information (and it should actually be using `PixelsPerOSPoint`).  But I guess that's not the purpose of this fix.

Reply:
Yeah, that feels out-of-scope for this fix. I would prefer to *just* fix the tests for now. We need a deep-dive into the TopFrameBackground-stuff soon anyway to deal with Android 8 issues.